------------------------------------------------------------
revno: 14959
revision-id: squid3@treenet.co.nz-20161226022824-yzvflwoj9boqjzbk
parent: squid3@treenet.co.nz-20161226021522-gh5su0gpeuhgn33n
committer: Amos Jeffries <squid3@treenet.co.nz>
branch nick: 4
timestamp: Mon 2016-12-26 15:28:24 +1300
message:
  Fix GCC6: unused local variable 'weInitiatedThisClosure'
------------------------------------------------------------
# Bazaar merge directive format 2 (Bazaar 0.90)
# revision_id: squid3@treenet.co.nz-20161226022824-yzvflwoj9boqjzbk
# target_branch: http://bzr.squid-cache.org/bzr/squid3/trunk/
# testament_sha1: 3a86b4644b5db283fbf1001843fbc13ae1128f67
# timestamp: 2016-12-26 02:51:07 +0000
# source_branch: http://bzr.squid-cache.org/bzr/squid3/4
# base_revision_id: squid3@treenet.co.nz-20161226021522-\
#   gh5su0gpeuhgn33n
# 
# Begin patch
=== modified file 'src/fs/rock/RockHeaderUpdater.cc'
--- a/src/fs/rock/RockHeaderUpdater.cc	2016-10-05 22:42:54 +0000
+++ b/src/fs/rock/RockHeaderUpdater.cc	2016-12-26 02:28:24 +0000
@@ -152,7 +152,7 @@
 Rock::HeaderUpdater::noteDoneReading(int errflag)
 {
     debugs(47, 5, errflag << " writer=" << writer);
-    if (const bool weInitiatedThisClosure = !reader) {
+    if (!reader) {
         Must(!errflag); // we only initiate successful closures
         Must(writer); // otherwise we would be done() and would not be called
     } else {

